app/helpers/application_helper.rb

```diff
@@ -56,4 +56,17 @@ def text(name)
     text = SiteText.where(name: name).first.try :text
     render_markdown text if text.present?
   end
+
+  def dashboard_path
+    # Just incase this method is called when a current user does not exist
```

Comment:
I was actually split on using that since I didn't know how the links should work when there's the "anonymous" login (as seen in the developer login), was planning to ask about it today. Good to know, I'll swap this.